keepers/validator-keeper/src/lib.rs

```diff
@@ -216,6 +244,7 @@ pub async fn emit_validator_history_metrics(
             get_vote_accounts_count,
             i64
         ),
+        ("num_get_vote_accounts_live_validators", get_vote_accounts_live_validators, i64),
```

Comment:
these names are a bit confusing, maybe 
`num_get_vote_accounts_voting`
`num_live_validator_histories`
open to other suggestions

Reply:
Done